source/help/apps/desktop-changelog.rst

```diff
@@ -1,6 +1,90 @@
 Desktop Application Changelog
 ========================================
 
+Release v4.7
+----------------------------
+
+**Download Binaries:** `Mattermost Desktop on GitHub <https://github.com/mattermost/desktop/releases/latest>`_
+
+**Release day TBD**
+
+Breaking Changes or Upgrade Notes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+- 
+
+Highlights
+~~~~~~~~~~~~~~~
+- Added support for BrowserView.
+
+Improvements
+~~~~~~~~~~~~~~~
+
+Windows
+^^^^^^^^^^^^^
+- Removed the dark mode toggle option for Windows.
```

Comment:
We should specify that dark mode now is controlled by the OS's default dark mode setting, so it doesn't look like we've removed functionality

Reply:
@amyblais "Windows desktop now automatically switches between light and dark themes based on the operating system settings."